### PR TITLE
Add 'Fees' and 'Fees (accumulated)' series to statement of assets series

### PR DESCRIPTION
**Pull request by OnkelDok**
Requested in forum here: https://forum.portfolio-performance.info/t/datenreihe-gebuehren-hinzufuegen/5389

Transactions with fees:
grafik

Resulting diagram:
grafik

Greetz

### CODE REVIEW

File: name.abuchen.portfolio/src/name/abuchen/portfolio/snapshot/ClientIndex.java [hunk 1]
```diff
@@ -183,9 +193,13 @@ private void collectTransferalsAndTaxes(Interval interval)
                                         break;
                                     case BUY:
                                         addValue(buys, t.getCurrencyCode(), t.getAmount(), interval, d);
+                                        Money feeBuy = t.getUnitSum(Type.FEE);
+                                        addValue(fees, feeBuy.getCurrencyCode(), feeBuy.getAmount(), interval, d);
```
Comment (buchen) [inline name.abuchen.portfolio/src/name/abuchen/portfolio/snapshot/ClientIndex.java:197]: Fees can also occur with deliveries (inbound and outbound).
I propose to handle the fees just like the tax in line 172 (see above):

```
// collect taxes
addValue(taxes, t.getCurrencyCode(), t.getUnitSum(Unit.Type.TAX).getAmount(), //
                                                interval, d);
```

**Comment by buchen** [comment]
I have applied the (small) change and merged your PR. Thanks 😄

File: name.abuchen.portfolio/src/name/abuchen/portfolio/snapshot/ClientIndex.java (hunk 1 above)
Comment (OnkelDok) [reply name.abuchen.portfolio/src/name/abuchen/portfolio/snapshot/ClientIndex.java:197]: Oh, I missed this type of transaction. Thank you for the hint and fixing it.

**Comment by OnkelDok** [comment]
> I have applied the (small) change and merged your PR. Thanks 😄

Thanks for your great work!

**Pull request closed without merging**